### DISCUSSION TITLE
Keep loosing your villagers?

### DISCUSSION THREAD

**Discussion by ChaliaManzotti** [Luke100000/minecraft-comes-alive#199] [Ideas]
With the new caves and cliffs update there is a lot of large....deep.. holes, I myself am finding half my village disappearing and only finding a few in the caves this makes it very hard to complete the average hearts quest to level up your village. I am proposing an item that sort of works how it does when there's a raid and you ring the bell being able to see the mobs through walls under a certain radius so then you can go out to find those lost villagers and take them back up to your village safely and block the danger to keep them from getting lost again.

Thank you for reading.

**Comment by frqnny** [comment]
Perhaps they scream "Help, I'm lost!" in the chat if they haven't been able to go to a home in a few MC days
